TradeItIosTicketSDK2/Info.plist

```diff
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.1.16</string>
```

Comment:
Could we use a variable (like ${CURRENT_PROJECT_VERSION} or customized variable), otherwise we will have to modify everywhere the version number ?